docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -440,19 +446,20 @@ grant and revoke this role. Extra precautions should be taken to secure
 accounts that have been granted it.
 
 [.contract-index#AccessControl-Mixin-Impl]
-.Embeddable Mixin Implementation
+.{mixin-impl}
 
 --
 .AccessControlMixinImpl
 
-* xref:#AccessControlComponent-Embeddable-Impls[`++AccessControlImpl++`]
-* xref:#AccessControlComponent-Embeddable-Impls[`++AccessControlCamelImpl++`]
+* xref:#AccessControlComponent-Embeddable-Impls-AccessControlImpl[`++AccessControlImpl++`]
+* xref:#AccessControlComponent-Embeddable-Impls-AccessControlCamelImpl[`++AccessControlCamelImpl++`]
 * xref:api/introspection.adoc#SRC5Component-Embeddable-Impls[`++SRC5Impl++`]
```

Comment:
i think it's fine, but note that this links to another section but we have a sub-index for src5 in this page too, that's left unlinked. no need to fix it i guess

Reply:
Yes, indeed. I thought it'd be bad UX to have links that point to other links that only point to another section (Introspection) with useful info, so this sort of eliminates the middleman while maintaining the API structure. It's not ideal though

Reply:
agree